Contrast-Security-OSS/go-test-bench
# Speed 628 standardize source input

Extract and standardize methods of getting user input from different sources. Refactor existing code in vulnerabilities and replace with the new approach - e.g. `r.URL.Query().Get("input")` gets replaced with `utils.GetQueryInput(r, utils.INPUT)`. Sources covered so far - **GET query** parameter, **GET URL** parameter, **POST body** parameter, **form** value, **cookie** value and **header** value.
 
How to test:
- run existing unit tests in util package, covering user input
- build the app and execute different vulnerabilities, their behaviour should remain the same (as if the app was run from the `main` branch)
- see vulnerabilities coming in TS - at this stage only post form data
